build_tools/cmake/iree_hal_cts_test_suite.cmake

```diff
@@ -116,6 +116,8 @@ function(iree_hal_cts_test_suite)
             "${IREE_ROOT_DIR}/iree/hal/cts/testdata/${_FILE_NAME}.mlir"
           FLAGS
             ${_TRANSLATE_FLAGS}
+          TRANSLATE_TOOL
+            "iree_tools_iree-translate"
```

Comment:
Which tool should this use? It uses `-iree-mlir-to-hal-executable`, not `-iree-mlir-to-vm-bytecode-module`.

Please also update the comments in this file to match (the find/replace changed to "iree-compile" in the comments, but this is actually using "iree-translate" in the implementation).

Reply:
iree-translate contains those test-only conversions.